### ISSUE TITLE
Riak fails with BadStatusLine

### ISSUE THREAD

**Issue by dcramer** [getsentry/sentry#1823]
https://app.getsentry.com/sentry/sentry/group/55181333/

```
BadStatusLine: ''
(1 additional frame(s) were not displayed)
...
  File "sentry/web/decorators.py", line 182, in _wrapped
    return func(request, *args, **kwargs)
  File "sentry/web/frontend/groups.py", line 548, in group_event_details_json
    Event.objects.bind_nodes([event], 'data')
  File "sentry/db/models/manager.py", line 251, in bind_nodes
    node_results = app.nodestore.get_multi(node_ids)
  File "sentry/nodestore/riak/backend.py", line 102, in get_multi
    six.reraise(type(err), err)
  File "<string>", line 2, in reraise
```

**Comment by mattrobenolt** [comment]
Switched over to using `pbc` protocol now, so this error will for sure go away, just not sure if it'll resurface as something different.

**Comment by mattrobenolt** [comment]
Switched over in production, will wait to claim victory.

**Comment by mattrobenolt** [comment]
Let's see if this is resolved with #1901.

**Comment by mattrobenolt** [comment]
Boom.
